docfx_yaml/extension.py

```diff
@@ -51,6 +51,7 @@
 from .monkeypatch import patch_docfields
 from .directives import RemarksDirective, TodoDirective
 from .nodes import remarks
```

Comment:
These imports will be refactored in a followup, and clean up methods that are not used.